Source/WebKit/WebProcess/WebPage/WebFrame.cpp

```diff
@@ -324,7 +324,7 @@ void WebFrame::didCommitLoadInAnotherProcess(WebCore::LayerHostingContextIdentif
     parent->tree().removeChild(*coreFrame);
     coreFrame->disconnectOwnerElement();
     auto client = makeUniqueRef<WebRemoteFrameClient>(*this, WTFMove(invalidator));
-
+    m_frameID = WebCore::FrameIdentifier(m_frameID.object(), processIdentifier);
```

Comment:
This is incorrect.  We want the FrameIdentifier to have the process identifier of the process that generated the identifier.

Reply:
Talking offline about how this makes sense, because that almost seems like redundant information to keep with the RemoteFrame. If it's required, I could investigate other ways of getting this information to RemoteFrame.